linode/manager
# upcoming: [M3-8311] - Auto-generate Linode Labels in Linode Create v2

## Description 📝

- Auto-generate Linode Labels on the client in Linode Create v2 🏭  
- This should work similarly to Linode Create v1 with the following differences
  - I changed the label generated on the backups tab to use the selected Linode's `label`  instead of the Linode's `region`. Region doesn't really make any sense, but I'm happy to change is back if we want parity 💾 
  - This implementation does not have logic to ensure the label is unique (will add in a follow up PR) 
  - Allows for generated labels to be longer because linode label max length was changed since this was originally implemented

## Preview 📷

https://github.com/user-attachments/assets/96573278-2fc4-4ea3-b8d2-84d3a5deb42b

## How to test 🧪

### Prerequisites
- Turn on Linode Create v2 feature flag 🎏 
- Ensure you are on Linode Create v2 by refreshing the page or remounting the Linode routes (this flag does not toggle immediately like our other flags)

### Verification steps
- Test the label generation on each tab of the Linode Create flow 📑 
- Verify the generated labels and valid, make sense with what you selected, and are similar to Linode Create v1 👀 

## As an Author I have considered 🤔
